static/webpack-config.js

```diff
@@ -33,6 +35,19 @@ module.exports = function () {
     globalConfig = parse(globalConfigRaw);
   }
 
+  const cssRtlPlugin = [];
```

Comment:
I agree with Connor using a list for a single item is strange, can't we just have it be
```js
let cssRtlPlugin;
//...
[
  cssRtlPlugin,
  ...htmlPlugins
]
```

Reply:
in the case that we don't need a cssRtlPlugin, it would have an undefined element in there and would throw an error when running webpack. I could move this chunk of code after `plugins` is first set and add to `plugins` directly if cssRtlPlugin is required, but then it might not be organize(?) as seeing everything that's suppose to be in plugins to be already there when it's first set.

Reply:
okay that's fair!

Reply:
Would renaming `htmlPlugins` to just `plugins` simplify things a little? then we could just push all plugins onto this array. We already have an array called `plugins` anyways.